ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -19,28 +19,73 @@ const LargeText = styled.div`
   font-size: 1.2em;
 `
 
-const UcscBrowserLink = ({ variant, useLiftover, includeEnd }) => {
-  const chrom = useLiftover ? variant.liftedOverChrom : variant.chrom
-  const pos = parseInt(useLiftover ? variant.liftedOverPos : variant.pos, 10)
-  let genomeVersion = useLiftover ? variant.liftedOverGenomeVersion : variant.genomeVersion
-  genomeVersion = genomeVersion === GENOME_VERSION_37 ? '19' : genomeVersion
+const UcscBrowserLink = ({ genomeVersion, chrom, pos, refLength, endOffset }) => {
```

Comment:
I made the logic for the link a bit simpler so it is passed the specific coordinates it needs, and moved the logic into the new `VariantPosition` component that parses out all the coordinates including the end and then renders the correct link(s)